src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -363,6 +373,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
       await Promise.all([
         AriesAgent.agent.connections.getUnhandledConnections(),
         AriesAgent.agent.credentials.getUnhandledCredentials(),
+        AriesAgent.agent.credentials.getUnhandledReqGrantEvents(),
```

Comment:
So this fetches them on start-up - do we also still receive new ones while the app is open?

Reply:
Yes, we still fetch the notifications continuously in the `onNotificationKeriStateChanged` function.